tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -1236,6 +1236,36 @@ async def validate_stop_on_wedge_point(self, bft_network, skvbc, fullWedge=False
                 with log.start_action(action_type='expect_kv_failure_due_to_wedge'):
                     with self.assertRaises(trio.TooSlowError):
                         await skvbc.write_known_kv()
+    
+    @with_trio
```

Comment:
You can set a default handler in kvbc/Replica.cpp for `setOnRestartProofCallBack` and then to make this test even better by showing that they restart themselves.